config/dev.exs

```diff
@@ -62,7 +62,8 @@ config :code_corps, CodeCorps.Mailer,
 config :code_corps,
   postmark_forgot_password_template: "123",
   postmark_project_acceptance_template: "123",
-  postmark_receipt_template: "123"
+  postmark_receipt_template: "123",
+  organization_invite_email_template: "123"
```

Comment:
In all of these there's a prefix of `postmark_`. We should keep that trend, and alpha-order accordingly.